cmd/daemon/events_linux.go

```diff
@@ -34,6 +34,7 @@ func (*dummyAnalytics) NotifyRequestAPI(events.DataRequestAPI) error   { return
 func (*dummyAnalytics) NotifyRate(events.ServerRating) error           { return nil }
 func (*dummyAnalytics) NotifyHeartBeat(int) error                      { return nil }
 func (*dummyAnalytics) NotifyLANDiscovery(bool) error                  { return nil }
+func (*dummyAnalytics) NotifyVirtualLocation(bool) error               { return nil }
```

Comment:
Is that also implemented on the libmoose side?

Reply:
Need to check if this setting is supported by moose and if it is will be implemented into a separate PR.

Reply:
I've created a ticket for this, because at the moment there is no event in moose.